spec/lib/blackout_updater_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe BlackoutUpdater do
+  describe 'update!' do
+    let!(:blackout) {instance_spy(Blackout, :update_attributes => nil)}
+    let!(:cf) {double("conflict", :md_link => "Test")}
```

Comment:
it's not as DRY, but it's preferred to not use `let!` and just declare the variables you need inside your tests. This way, every individual test is readable and doesn't depend on things declared elsewhere.